src/update-changelog.ts

```diff
@@ -213,10 +213,12 @@ async function runCommand(command: string, args: string[]): Promise<string[]> {
  * @param description - The commit message description.
  * @returns The category of the change.
  */
-function getCategory(description: string): ChangeCategory {
-  // Check if description contains a colon
-  if (description.includes(':')) {
-    const [prefix] = description.split(':').map((part) => part.trim());
+export function getCategory(description: string): ChangeCategory {
+  const conventionalCommitPattern = /^(feat|fix)(?:\([^)]*\))?\s*:\s*/u;
+  const match = description.match(conventionalCommitPattern);
+
+  if (match) {
+    const prefix = match ? match[1] : undefined;
```

Comment:
Since we already check for `match` being nullish in the previous line, perhaps this should be:

```suggestion
    const prefix = match.length ? match[1] : undefined;
```